server/app/api.py

```diff
@@ -889,6 +891,14 @@ def queues(self, obj, user, data):
 
         return models.Queue.query(models.Queue.assignment == obj.key).fetch()
 
+    def statistics(self, assignment, user, data):
+        """Returns assignment statistics"""
```

Comment:
How are permissions handled? 
We could add permissions checking here - Need('Staff') (see line 888 above)